Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -996,13 +996,14 @@ def rasterize_email_command():  # pragma: no cover
 
     navigation_timeout = int(demisto.args().get('max_page_load_time', DEFAULT_PAGE_LOAD_TIME))
 
-    with open('htmlBody.html', 'w', encoding='utf-8-sig') as f:
-        f.write(f'<html style="background:white";>{html_body}</html>')
-
-    path = f'file://{os.path.realpath(f.name)}'
-
-    rasterize_output = perform_rasterize(path=path, rasterize_type=rasterize_type, width=width, height=height,
-                                         offline_mode=offline, navigation_timeout=navigation_timeout, full_screen=full_screen)
+    with tempfile.NamedTemporaryFile(mode='w', suffix='.html', encoding='utf-8-sig') as tf:
+        demisto.debug(f'{html_body=}')
+        tf.write(f'<html style="background:white";>{html_body}</html>')
```

Comment:
if from some reason this line will fail i think that the tempfile will not being closed properly. maybe it is better to wrap this with try, except, finally tf.close()

Reply:
Hey @ilaredo 
In case write to the file will fail the file will be automatically closed and deleted once we get out form the context manager (this is the idea of the [tempFile](https://docs.python.org/3/library/tempfile.html#module-tempfile))

Reply:
Hey @ilaredo 
In case write to the file will fail the file will be automatically deleted once we exit the context manager
This is the idea behind the [tempFile](https://docs.python.org/3/library/tempfile.html#module-tempfile)